src/Main.elm

```diff
@@ -52,51 +52,129 @@ type DecimalStatus
 executeAdd : Model -> Model
 executeAdd model =
     { model
-        | display = String.fromFloat <| model.valueA + model.valueB
+        | display = showResults <| model.valueA + model.valueB
         , valueA = 0
-        , actionA = Blank
+        , actionA = Nothing
         , valueB = 0
-        , actionB = Blank
+        , actionB = Nothing
         , displayStatus = ShowingResults
     }
 
 
+executeSubtract : Model -> Model
+executeSubtract model =
+    { model
+        | display = String.fromFloat <| model.valueA - model.valueB
+        , valueA = 0
+        , actionA = Nothing
+        , valueB = 0
+        , actionB = Nothing
+        , displayStatus = ShowingResults
+    }
+
+
+executeMultiply : Model -> Model
+executeMultiply model =
+    { model
+        | display = String.fromFloat <| model.valueA * model.valueB
+        , valueA = 0
+        , actionA = Nothing
+        , valueB = 0
+        , actionB = Nothing
+        , displayStatus = ShowingResults
+    }
+
+
+executeDivide : Model -> Model
+executeDivide model =
+    { model
+        | display = String.fromFloat <| model.valueA / model.valueB
+        , valueA = 0
+        , actionA = Nothing
+        , valueB = 0
+        , actionB = Nothing
+        , displayStatus = ShowingResults
+    }
+
+
+showResults : Float -> String
+showResults float =
+    let
+        string =
+            String.fromFloat float
+
+        hasDecimal =
+            String.any (\a -> '.' == a) string
+
+        largerThenTen =
+            case hasDecimal of
+                True ->
+                    String.length string > 11
+
+                False ->
+                    String.length string > 10
+    in
+    case ( string, largerThenTen ) of
```

Comment:
This could just be an `if` on `largerThenTen` (should be `largerThanTen` btw).

You're not actually doing any pattern matching on the `string` variable, so no need to include it in the `case`

Reply:
removed unused string and kept is as a `case`.